front/src/applications/rollingStockEditor/components/RollingStockEditorFormHelpers.tsx

```diff
@@ -151,7 +151,7 @@ const RollingStockEditorParameterFormColumn = ({
               handleType={(type) => {
                 setRollingStockValues({
                   ...rollingStockValues,
-                  [property.title]: type.value,
+                  [property.title]: Number(type.value) || 0,
```

Comment:
What's this line fixing ? I can't speak front-end :D

Reply:
Type.value is a string here, we need to convert it to a number, otherwise the backend can not accept the request :) Now, you can edit the resistance fields, and everything should work correctly !